src/sass/_nav.scss

```diff
@@ -286,3 +286,15 @@ sub {
         }
     }
 }
+@media screen and (max-width: 1190px) {
```

Comment:
There should be a min-width so the icons are not hidden in smaller screens inside the hamburger menu. Also, we should hide all the icons at once, using the .icon selector.